core/src/main/scala/org/apache/spark/internal/config/Deploy.scala

```diff
@@ -82,4 +82,9 @@ private[spark] object Deploy {
     .checkValue(_ > 0, "The maximum number of running drivers should be positive.")
     .createWithDefault(Int.MaxValue)
 
+  val DRIVER_ID_PATTERN = ConfigBuilder("spark.deploy.driverIdPattern")
+    .doc("The pattern for driver ID generation.")
```

Comment:
Can we document the pattern parameters too?

Reply:
Sure!